src/ontology/cl-edit.owl

```diff
@@ -30612,6 +30613,20 @@ SubClassOf(obo:CL_4030037 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_000007
 SubClassOf(obo:CL_4030037 ObjectSomeValuesFrom(obo:RO_0002202 obo:CL_4030036))
 SubClassOf(obo:CL_4030037 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0007286))
 
+# Class: obo:CL_4030038 (CD24-positive, CD-133-positive, vimentin-positive proximal tubular cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:21281815") Annotation(oboInOwl:hasDbXref "PMID:23124355") obo:IAO_0000115 obo:CL_4030038 "A CD24-positive, CD-133-positive, vimentin-positive cell found scattered throughout a renal proximal tubule and that may participate in tubular regeneration. Compared to other proximal tubular cell types, this cell contains less cytoplasm, fewer mitochondria and no brush border.")
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4030038 "2023-03-07T14:17:39Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "https://doi.org/10.1002/stem.1130") oboInOwl:hasBroadSynonym obo:CL_4030038 "CD133+CD24+CD106- tubular cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:23124355") oboInOwl:hasBroadSynonym obo:CL_4030038 "CD24-positive tubular cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:23124355") oboInOwl:hasBroadSynonym obo:CL_4030038 "CD24+CD133+ tubular cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:23124355") rdfs:comment obo:CL_4030038 "After acute tubular injury, this cell type becomes more numerous. In healthy rat kidneys, this cell is not detectable, whereas after unilateral ureteral obstruction (renal injury), it may appear de novo, arguing against the notion that this cell type represents a pre-existing progenitor cell population. In at least one study, data indicate that this cell type represents transiently dedifferentiated tubular cells involved in regeneration.")
+AnnotationAssertion(rdfs:label obo:CL_4030038 "CD24-positive, CD-133-positive, vimentin-positive proximal tubular cell")
+SubClassOf(obo:CL_4030038 obo:CL_0002306)
+SubClassOf(obo:CL_4030038 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0008404))
```

Comment:
I am not sure why this is not inferred, as an epithelial cell of proximal tubule I would expect to be part of some proximal tubule epithelium

Reply:
At first glance, it may be because `'part of' some 'proximal tubule epithelium'` is not asserted on 'epithelial cell of proximal tubule'.